src/components/utrecht/FormField.test.jsx

```diff
@@ -0,0 +1,69 @@
+import { render, screen } from "@testing-library/react";
+import { createRef } from "react";
+import { FormField } from "./FormField";
+import "@testing-library/jest-dom";
+
+describe("Form field", () => {
+  it("renders an HTML div element", () => {
+    const { container } = render(<FormField></FormField>);
```

Comment:
Can we make this a self closing component as we use those in other tests as well?